.gitignore

```diff
@@ -106,6 +106,10 @@ build-install/*
 .cache/*
 .pytest_cache/*
 
+# hypothesis cache #
+####################
+**/.hypothesis
+
```

Comment:
I think this can be removed because the Hypothesis home directory is set by the profile to a temp directory.
```suggestion
```

Reply:
indeed